oracle/weblogic-image-tool
# Patchverify

There are substantial changes in this commit:

1.  Remove online md5 verification test for now
2.  All patchId format is now with p in the front.   The format is 99999999_1.2.3.4.567890  patchid followed by optional release string.  The release string is what shows up in the release column when searching for patches in support.oracle.com [not to be confused with anything in the filename]
3.  In create if only the base patch bug number is specified and multiples are found and it will fail with messages listing what are the valid values for that patch. It is a short circuit failure.
4.  Add patch conflicts checks in create image
5.  update will now use the actual release number of the patch for conflicts check
6.  Remove obsolete arguments for ```--version``` ```-user``` ```-password*``` in ```cache addPatch``` command, only the ```patchId``` is used for the key,  there are minimal regex check for the format.
7.  Updated the docs to reflect the changes to patch id format and cache commands
